Src/library/base/elks/kernel/file_name.e

```diff
@@ -24,6 +26,11 @@ feature {NONE} -- Initialization
 
 	make_temporary_name
 			-- Create a temporary filename.
+		note
+				EIS:"name={FILE}.make_temporary_open", "src=eiffel:?class=FILE&feature=make_temporary_open","protocol=uri"
+				EIS:"name={FILE}.make_temporary_open_with_prefix", "src=eiffel:?class=FILE&feature=make_temporary_open_with_prefix","protocol=uri"
+		obsolete
+				"Use `{FILE}.make_temporary_open or {FILE}.make_temporary_open_with_prefix' instead. [2019-05-31]"
```

Comment:
you should use the new convention with two back-ticks `foo` instead of back-tick and quote `foo'

Reply:
Is there a way to implement FILE_NAME.make_temporary_name without tempnam ?
If yes, even if this is not 100% safe, it would be good to have it.

Otherwise we'll still have the tempnam C warning for a few releases.

Reply:
Need to be checked, but that's the idea to make it obsolete.